packages/litexa/src/command-line/deploy.coffee

```diff
@@ -113,44 +120,50 @@ module.exports.run = (options) ->
     if deploymentTypes.assets
       assetsLogger = new LoggingChannel({
         logPrefix: 'assets'
-        logStream
-        logFile: path.join(deployRoot,'assets.log')
+        logSteam: logger.logStream
```

Comment:
logSteam typo > applies to below as well.

Reply:
good catch! Fixed